stable/sysdig/README.md

```diff
@@ -92,6 +94,30 @@ $ helm install --name sysdig-agent-on-prem \
     stable/sysdig
 ```
 
+## Using private image registries
+
+To authenticate against an image registry you will need to store the credentials
+in a ConfigMap and add a reference to that ConfigMap.
```

Comment:
To authenticate against a private image registry you will need to store the credentials
in a Secret and the Helm chart will add a reference to that Secret in the generated ConfigMap.